<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageUsersDialog</class>
 <widget class="QDialog" name="ManageUsersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Users Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableView" name="view">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addUserButton">
       <property name="text">
        <string>Add User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeUserButton">
       <property name="text">
        <string>Remove User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setPasswordButton">
       <property name="text">
        <string>Set Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeGroupButton">
       <property name="text">
        <string>Change group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeDescriptionButton">
       <property name="text">
        <string>Change description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateListButton">
       <property name="text">
        <string>Update list</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>addUserButton</tabstop>
  <tabstop>removeUserButton</tabstop>
  <tabstop>setPasswordButton</tabstop>
  <tabstop>changeGroupButton</tabstop>
  <tabstop>changeDescriptionButton</tabstop>
  <tabstop>updateListButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
